provider/resource_keycloak_openid_client_authorization_permission.go

```diff
@@ -59,6 +60,10 @@ func resourceKeycloakOpenidClientAuthorizationPermission() *schema.Resource {
 				Elem:     &schema.Schema{Type: schema.TypeString},
 				Optional: true,
 			},
+			"resource_type": {
+				Type:     schema.TypeString,
+				Optional: true,
```

Comment:
In the GUI, it looks like you can't set the resource type and a list of resources.  Should we add a `ConflictsWith` here and within the `resources` attribute?

Reply:
I went ahead and pushed this change to your branch so I could get this merged / released today.  If it turns out you don't think this is needed, we can remove it in a follow-up PR.

Reply:
Ah, thanks. I had that change being made but was running acceptance tests! Sorry about the delay getting back to you, it's been a crazy morning.

Reply:
You have nothing to apologize for, I'm pretty sure my delay in reviewing this PR was longer 😉